app/models/statistics_announcement.rb

```diff
@@ -46,6 +46,7 @@ def can_publish_to_publishing_api?
               in: PublicationType.statistical.map(&:id),
               message: "must be a statistical type",
             }
+  validates :publication_id, uniqueness: true, if: -> { publication_id.present? }
```

Comment:
Should it be possible for a `publication_id` to not be present? Or should we enforce its presence?

If `StatisticsAnnouncement belongs_to :publication` and `Publication has_one :statistics_announcement`, why would we allow `publication_id` to be `nil`/`NULL`?

Reply:
It seems like it's quite common for statistics announcements not to be assigned to any publication; this is from integration:

```
irb(main):002:0> StatisticsAnnouncement.where(publication_id: nil).count
=> 2889
```

Reply:
Ok, then if we're ok with there being `nil` values, I'd suggest being explicit about it by changing `belongs_to :publication` to `belongs_to :publication, optional: true`.